hq/app/logic/VulnerableAccessKeys.scala

```diff
@@ -1,33 +1,42 @@
 package logic
 
 import config.Config.{iamHumanUserRotationCadence, iamMachineUserRotationCadence}
-import model.{AccessKey, AccessKeyEnabled, VulnerableAccessKey}
+import model.{AccessKey, AccessKeyEnabled}
 
 object VulnerableAccessKeys {
```

Comment:
Only a couple of methods in this object are used and they're used for dashboard in `CredentialReportDisplay`. If we don't remove this unused code now, let's make sure to do it later.